singlestoredb/management/job.py

```diff
@@ -130,13 +134,13 @@ def from_dict(cls, obj: Dict[str, Any]) -> 'ExecutionConfig':
 class Schedule(object):
 
     execution_interval_in_minutes: int
-    start_at: datetime.datetime
+    start_at: Union[str, datetime.datetime]
```

Comment:
Isn't this always a datetime? It's ok if the constructor accepts a string or datetime (that would be good actually) which gets converted to a datetime in the constructor, but allowing either value type makes it harder for other code to use this variable.

Reply:
Honestly, I only went with the `Union[str, datetime.datetime]` because I was having issues with `pre-commit` checks on `from_dict` complaining that `from_datetime` might return `None`.
I found no other solution, but if you have any suggestions I'll be happy to change this.

Reply:
Probably the best way to fix this would be to change it to datetime and add a `to_datetime_strict` function that only returns a datetime or raises an exception (no `None` possible), then use that function to cast the value.

Reply:
@kesmit13 I did as you suggested, but I also fixed a bug in the implementation of `to_datetime` where we were never including the hour, mins, seconds, etc because `datetime` is an instance of `date`.
I don't know if this might be a breaking change for any users that are doing logic on top of any returned `datetime`.